particle/particle/particle.py

```diff
@@ -148,22 +148,22 @@ class Particle(object):
     pdgid = attr.ib(converter=PDGID)
     pdg_name = attr.ib()
     mass = attr.ib()
-    width = attr.ib()
-    anti_flag = attr.ib(converter=Inv)  # Info about particle name for anti-particles
-    _three_charge = attr.ib(Charge.u, converter=Charge)  # charge * 3
-    rank = attr.ib(0)  # Next line is Isospin
+    mass_upper = attr.ib(0.0)
+    mass_lower = attr.ib(0.0)
+    width = attr.ib(-1.)
+    width_upper = attr.ib(0.0)
+    width_lower = attr.ib(0.0)
+    Q_times_3 = attr.ib(Charge.u, converter=Charge)  # charge * 3
```

Comment:
Since we use charge instead of Q, shouldn't this be charge_times_3?

Reply:
I will revert back to `_three_charge` as we already have the `three_charge` property as part of the public API. My goal here was to produce a table with nice names, but that can be easily achieve another way.